### PR TITLE
Remove the Apply With Renaming option.

### PR DESCRIPTION
**Pull request by ppedrot**
It was deprecated since Coq 8.15.

### CODE REVIEW

**Comment by ppedrot** [comment]
@coqbot run full ci

**Comment by ppedrot** [comment]
Any taker? @SkySkimmer maybe?

File: plugins/funind/recdef.ml [hunk 1]
```diff
@@ -105,12 +105,7 @@ let next_ident_away_in_goal ids avoid =
 
 let compute_renamed_type gls id =
   let t = Tacmach.pf_get_hyp_typ id gls in
-  if Clenv.rename_with () then
-    rename_bound_vars_as_displayed (Global.env ()) (Proofview.Goal.sigma gls)
-      (*no avoid*) Id.Set.empty (*no rels*) []
-      t
-  else
-    t
+  t
```
Comment (silene) [inline plugins/funind/recdef.ml:108]: I think you should go one step further and replace both occurrences of `compute_renamed_type` with `Tacmach.pf_get_hyp_typ`. As it stands, the function no longer does what its name says it does.

Other than that, it looks good to me, assuming the VST failure is unrelated.

**Review by silene — APPROVED** [review]

**Review by SkySkimmer — APPROVED** [review]

**Comment by ppedrot** [comment]
No need to run a full CI (which will fail anyways because there are missing overlays), because it was already run before @silene asked for a minor cosmetic change in funind.

**Comment by SkySkimmer** [comment]
@coqbot merge now

**Pull request merged**